Update repository documentation and training exercises (#394)
ros-industrial/industrial_training: gh_pages/_source/session3/ros2/3-Build-a-MoveIt-Package.md

    ompl_planning.yaml
    ```

>  1. Open the ROS2 `config/ompl_planning.yaml` file in a text editor and add the following lines at the bottom, with no indentation:

Reviewer: The "with no indentation" piece could be confusing - we do need some of these lines to have indentation.  Do you feel like "Make sure your editor does not add extra whitespace to the front of these lines" would have added the clarity you needed?

        default_planner_request_adapters/FixStartStatePathConstraints
    start_state_max_bounds_error: 0.1
    ```
    

>     Make sure your editor does not add extra whitespace to the front of these lines.

Reviewer: ```suggestion
    Make sure your editor does not change the amount of whitespace at the front of these lines.
```
